packages/components/src/core/styles/common/defaultTheme.ts

```diff
@@ -239,44 +252,193 @@ export const defaultAppTheme: AppTheme = {
   },
 };
 
+const lightAppTheme: AppTheme = {
+  colors: lightThemeColors,
+  ...sharedAppTheme,
```

Comment:
should we swap the order here, so we ensure `colors: lightThemeColors,` will overwrite `sharedAppTheme.colors` if `sharedAppTheme` happens to have `colors` at runtime 😄 ?

Reply:
Good point! Even though I've said `sharedAppTheme` omits `colors`, that doesn't mean it'll actually happen.

The joys of TypeScript lol

Swapping!